### DISCUSSION TITLE
Add setting configuration to datasource

### DISCUSSION THREAD

**Discussion by JackyWoo** [grafana/grafana#37922] [Data Sources]
#### Motivation
I use grafna with MySQL DataSource and I use 'group_concat' function which has a result longer than default value 1024. So I want to change the setting 'group_concat_max_len'. But I can not find out the configuration. 

I think maybe there are other cases like that. So add setting configuration to datasource maybe useful.

**Comment by gabor** [comment]
hi @JackyWoo , based on https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_group_concat_max_len, this is a system variable, i recommend setting it directly in the configuration of your mysql database ( not in the mysql-datasoure-plugin's config page).

**Comment by gabor** [comment]
additionally, we are not using github discussions for feature requests anymore, so , if you think my recommendation does not solve your problem, please create a new github issue, and describe there why this solution is not sufficient. thanks!

**Reply by JackyWoo** [reply]
hi @gabor Thanks for your advice. But I think it will be better for mysql-datasoure-plugin can support the setting for the mysql server is administrated by another team. I will submit an issue. Thanks again.
